CONTRIBUTING.md

```diff
@@ -135,3 +135,23 @@ able to be built and passes all checks performed by CI.
 Move uses [GitHub issues](https://github.com/move-language/move/issues) to track
 bugs. Please include necessary information and instructions to reproduce your
 issue.
+
+## Major feature requests
+
+Please begin by checking the [language feature request graveyard](GRAVEYARD.md) to ensure that your idea is not present.
```

Comment:
Shouldn't we also mention (resp. give a search URL) for existing proposed and accepted features so we don't get duplicates?

Reply:
Good call, will add links to the issue board filtered by the appropriate tags.